packages/core/.storybook/preview-body.html

```diff
@@ -1,2 +1 @@
 <div id="dropdown-portal"></div>
-<div id="tooltips-container"></div>
```

Comment:
Leading principle: remove the need to render floatings (tooltips, tipseens, dialogs) in a certain container and render them in the body (with z-index givent to the `monday-style-dialog-content-wrapper` class which is used in DialogContent), unless specifically given "containerSelector" which will adjust the rendering to that node. On top of modals they will be rendered in the modal's node, meaning same z-index as the modal.

Reply:
In a different PR we can export the layer provider along with instructions how to use it elsewhere